submit_message.php

```diff
@@ -0,0 +1,47 @@
+<?php session_start(); ?>
+<?php
+if (!isset($_SESSION["username"])) {
+    header('Location: login.php');
+}
+?>
+
+<html lang="en">
+<body>
+
+
+<?php
+$servername = "localhost";
+$username = "root";
+$password = "";
+
```

Comment:
People used to like to take those variables into a different file and include it ,  just so you know .

Reply:
Thank you!